scripts/geocode_addresses.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+"""
+Geocode polling addresses from a CSV input file. Adds columns for the clean
+address, latitude, and longitude for each row and writes the results to a new CSV file.
+
+Usage:
+    geocode_addresses.py <infile> <outfile>
+    geocode_addresses.py -h | --help
+
+Arguments:
+    <infile>  The CSV file to use an input.
+    <outfile>  The location to write an output CSV file. If an existing file exists it will be overwritten.
+"""
+
+import csv
+import os
+import googlemaps
+from docopt import docopt
+
+gmaps = googlemaps.Client(key=os.getenv('GOOGLE_API_KEY'))
```

Comment:
We've been slowly working our way away from using Google's API, and also I think it'd be nicer to use a free resource if possible and avoid having the register/handle API keys. 🙈 Have you considered the [`geocoder` Python library](https://geocoder.readthedocs.io/index.html)? I notice it happens to have an interface to 
[OpenStreetMap addresses via Nominatum](https://geocoder.readthedocs.io/providers/OpenStreetMap.html).

Reply:
I haven't looked at that, the ticket just described using Google Maps, but I certainly can. A few things to consider at first glance:

1. It looks like we might have to run our own server to support that
2. The Google results provide us with a "formatted address" that we are relying on. At a glance, I haven't yet seen support for that in this library (but I very well just might not have come across it yet).
3. This is something we'll just be running manually once every 2 years, so the requirements about keys & free feel a little less onerous. Given that I already ran this and got the results for this year, checking in this script is basically about making it easier next time so we don't have to figure everything all out from scratch again. So we might want to consider upgrades like a new library in a separate ticket.